firefly-demo.sh

```diff
@@ -3,7 +3,7 @@
 set -euo pipefail
 
 # contact Web API and download runner
-curl http://firefly-test.cvlad.info/script?firefly_token="NVdnOFRscE5BWWFRRXZmVnA1bTVFc09RZXJGNkxqZnJkQVQ4bHljWHZRdz0=" | sh
```

Comment:
Would it make sense to do something like `FIREFLY_TOKEN="NVdnOFRscE5BWWFRRXZmVnA1bTVFc09RZXJGNkxqZnJkQVQ4bHljWHZRdz0="` and use that here and below?

Reply:
Changed